LibraBFT/Prelude.agda

```diff
@@ -327,3 +327,10 @@ module LibraBFT.Prelude where
   f-sum f = sum ∘ List-map f
 
   open import LibraBFT.Base.Util public
+
+  -- HC-TODO : Lemmas has an isJust - should it be renamed, or this one?
+  isjust : ∀{ℓ}{A : Set ℓ} → Maybe A -> Bool
```

Comment:
This is `is-just` in the Agda standard library.

Reply:
I removed this def and used `is-just`.